src/main/java/uk/gov/justice/hmpps/prison/service/OffenderDamageObligationService.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.justice.hmpps.prison.service;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.hmpps.prison.api.model.OffenderDamageObligationModel;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderDamageObligation;
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderDamageObligationRepository;
+import uk.gov.justice.hmpps.prison.security.VerifyOffenderAccess;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class OffenderDamageObligationService {
+    private final OffenderDamageObligationRepository repository;
+    private final String currency;
+
+    public OffenderDamageObligationService(@Value("${api.currency:GBP}") final String currency, final OffenderDamageObligationRepository repository) {
```

Comment:
not sure why currency is a constructor parameter - can't see it ever changing.  Better to just use field initialisation